modules/cluster-logging-rn-5.7.0.adoc

```diff
@@ -0,0 +1,22 @@
+//module included in cluster-logging-release-notes.adoc
+:content-type: REFERENCE
+[id="cluster-logging-release-notes-5-7-0{context}"]
+= Logging 5.7.0
+This release includes link:https://access.redhat.com/errata/RHBA-2023:2133[OpenShift Logging Bug Fix Release 5.7.0].
+
+[id="openshift-logging-5-7-enhancements"]
+== Enhancements
+With this update, you can enable logging to detect multi-line exceptions and reassemble them into a single log entry.
+
+[id="openshift-logging-5-7-known-issues"]
+== Known Issues
+None.
+
+[id="openshift-logging-5-7-0-bug-fixes"]
+== Bug fixes
+* Before this update, the `nodeSelector` attribute for the Gateway component of the LokiStack did not have any effect. With this update, the `nodeSelector` attribute works as expected. (link:https://issues.redhat.com/browse/LOG-3713[LOG-3713])
```

Comment:
Can we elaborate technically on what does the phrase "did not have any effect" means in this sentence? For example, did it not effect the operations of the attribute or something else?